### DISCUSSION TITLE
multiple interval selection

### DISCUSSION THREAD

**Discussion by rhanumante** [vega/vega-lite#8498] [Q&A]
Hi community, How we can do multiple interval selection?

**Comment by jwoLondon** [comment]
There's an example of this in the documentation: https://vega.github.io/vega-lite/docs/selection.html#resolve 

If you select the `intersect` example you can apply multiple interval selections and highlight, for example, just the intersecting points. 

Is that enough to work with?

**Reply by rhanumante** [reply]
Hi JwoLondon,

Thanks for the reply, Not working for me.

"selection": {
                "brush": {
                    "type": "interval",
                    "encodings": [
                        "x"
                    ],
                    "on": "[mousedown, window:mouseup] > window:mousemove!",
                    "translate": "[mousedown, window:mouseup] > window:mousemove!",
                    "zoom": "wheel!",
                    "mark": {
                        "fill": "#333",
                        "fillOpacity": 0.125,
                        "stroke": "white"
                    },
                    "resolve": "intersect"
                }
            },

Actually I want to achieve [this](https://i.stack.imgur.com/RAsqv.png) kind of functionality. If you have some sample example like my use case then it will be great help. 

on the left side there is the table from that user can add x axis distance from the marker as the start point and then accordingly selection will be done on the chart as you can see in the URL given. Also, I want to get the selection values for the x axis when we select from the chart. 

Thanks again.
